R/zzz.R

```diff
@@ -14,9 +14,17 @@ lint_inputs <- function(lon = NULL, lat = NULL, format) {
 }
 
 # FIXME: stopgap for now until figure out how to replace these on on src side
-# - smart quote
-# - degree symbol
-# - masculine oridinal indicator
-scrub <- function(x) gsub("\u2019|\u00b0|\u00ba", "'", x)
+# - single quotes
+# - double quotes
+# - degree symbols
+# - masculine ordinal indicator
+# - rare separators
+scrub <- function(x) {
+  if(is.character(x)){
+    if(length(x) == 1 && Encoding(x) != "UTF-8") x <- iconv(x, to = 'UTF-8')
```

Comment:
i'm not sure I follow why there are two if/else lines here. that is, it seems like length of the vector shouldn't matter, or am I wrong?

Reply:
Right, I committed a shorter and more straightforward encoding conversion.

Reply:
thanks